temporal-serviceclient/src/main/java/io/temporal/serviceclient/ChannelManager.java

```diff
@@ -140,7 +145,9 @@ private Channel applyHeadStandardInterceptors(Channel channel) {
     headers.put(CLIENT_NAME_HEADER_KEY, CLIENT_NAME_HEADER_VALUE);
 
     return ClientInterceptors.intercept(
-        channel, MetadataUtils.newAttachHeadersInterceptor(headers));
+        channel,
+        MetadataUtils.newAttachHeadersInterceptor(headers),
+        new SystemInfoInterceptor(systemCapabilitiesFuture));
```

Comment:
Just double checking that these two will go first before any user-specified interceptors

Reply:
yup